NMOSUtils.py

```diff
@@ -56,6 +56,10 @@
     (63072000, 63072009),  # 1 Jan 1972, 10 leap seconds
 ]
 
+IMMEDIATE_ACTIVATION = 'activate_immediate'
```

Comment:
Looks like the old definition (and usage) of these in the Activation class is still present. I'm guessing you meant to refactor this?

Reply:
Fixed